test/claims-status/components/appeals-v2/Issues.unit.spec.jsx

```diff
@@ -21,8 +22,13 @@ describe('<Issues/>', () => {
     expect(wrapper.find('CollapsiblePanel').length).to.equal(0);
   });
 
-  it('should render one panel when only an open *OR* a closed issue is passed in', () => {
-    const wrapper = shallow(<Issues {...oneIssues}/>);
+  it('should render one panel when only an open issue is passed in', () => {
+    const wrapper = shallow(<Issues {...oneOpenIssue}/>);
+    expect(wrapper.find('CollapsiblePanel').length).to.equal(1);
+  });
+
+  it('should render one panel when only a closed issue is passed in', () => {
+    const wrapper = shallow(<Issues {...oneClosedIssue}/>);
     expect(wrapper.find('CollapsiblePanel').length).to.equal(1);
```

Comment:
Same here as above.

Reply:
Yeah that's fine; these don't have any special identifiers but I can dive into them and then inspect the panel names